src/pages/application_status/index.tsx

```diff
@@ -0,0 +1,373 @@
+import { useState } from 'react'
+import Head from 'next/head'
+import {
+  Button,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@chakra-ui/react'
+import Image from 'next/image'
+import TopNavbar from '~/components/common/TopNavbar'
+import withAuth, { BaseProps } from '~/utils/withAuth'
+
+enum PageState {
+  LOGIN,
+  FORGET_PASSWORD,
+}
+
+const Application_Status: React.FC<BaseProps> = ({ session }) => {
+  //this part needs backend to update the status
+  const [isOpen, setIsOpen] = useState(false)
+  const [isAcceptOpen, setIsAcceptOpen] = useState(false)
+  const [isRejectOpen, setIsRejectOpen] = useState(false)
+  const onOpenAccept = () => setIsAcceptOpen(true)
+  const onCloseAccept = () => setIsAcceptOpen(false)
+  const onOpenReject = () => setIsRejectOpen(true)
+  const onCloseReject = () => setIsRejectOpen(false)
```

Comment:
To make this neat, maybe can be like:

```suggestion
const [status, setStatus] = useState({
  accept: false,
  reject: false
})
```

And when you want to toggle one, you can be like

```suggestion
setStatus({
  ...status,
  accepted: !status.accepted
})
```

And likewise the opposite